e2e/js/js-cdn/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
there should be a github action somewhere to run this on PR

Reply:
I think its handled already? I dont see any specific actions for linting so I assume yarn affected:lint catches it

Reply:
by "this" I meant the e2e test

Reply:
oh lol. Well that's already handled by yarn affected:e2e as well

Reply:
Ill have to ensure it properly handles the impliedDependencies like you mentioned in another comment